src/wp-includes/class-wp-block-type.php

```diff
@@ -398,6 +402,10 @@ public function __isset( $name ) {
 			return true;
 		}
 
+		if ( 'uses_context' === $name ) {
```

Comment:
I feel we should avoid adding separate conditions for look-alike ones.
Instead, club both `variations` and `uses_context` checks in one condition using `in_array`

Reply:
Sure! I just did it in [this commit](https://github.com/WordPress/wordpress-develop/pull/6079/commits/f4af4f7f794f47822b2335f779b790404317d1bd).